src/components/atoms/Buttons.tsx

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
+import React, {FC} from 'react';
 import {
   StyleSheet,
   Text,
   TouchableWithoutFeedback,
   View,
 } from 'react-native';
 
-export const Button = (props) => {
+interface Button {
+  title: string;
+  onPress: () => void;
+  style: {};
+  disabled: boolean;
+}
+export const Buttons: FC<Button> = (props) => {
   return (
     <TouchableWithoutFeedback onPress={props.onPress} disabled={props.disabled}>
       <View style={[styles.button, {...props.style}]}>
```

Comment:
everything must be a styled component.. no adding styles in tsx... your component must have folder ex: /Button
inside that index.tsx and styles.ts.. all the styled components in styles.ts